### DISCUSSION TITLE
Can't Reverify for Student Pack

### DISCUSSION THREAD

**Discussion by ZackRuybal** [community/community#56158] [GitHub Education]
### Hi everyone,

It seems like I'm in the same boat as a lot of other people here that can no longer re-verify their credentials for the student pack. I'm not sure what's changed since the last time I did this in 2021, but this new system seems to be pretty terrible. My name is clearly on the ID, as well as on my Github profile, and I've got a short bio, so I'm really not sure why I keep getting this message. This is pretty frustrating.

<img width="699" alt="image" src="https://github.com/community/community/assets/8094568/6fe51cc9-3953-4d72-95f7-cc874f982827">

**Comment by donggook-me** [comment]
I am in the same trouble. But the problem is, after github rejected first application, I cannot use any other credentials.. I don't have anything else then..

**Comment by samus-aran** [comment]
Hi @ZackRuybal 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈  I suspect this will help if you have already updated your profile.

Hope this helps 👍
